### ISSUE TITLE
'MSELoss' object has no attribute 'next_functions'

### ISSUE THREAD

**Issue by sydney0zq** [pytorch/tutorials#97]
I am reading tutorial of learning `pytorch`, but I get stuck in one error as follow.

<https://github.com/pytorch/tutorials/blob/master/beginner_source/blitz/neural_networks_tutorial.py>

Lines from 184 to 186 are my source code. And I have replace `grad_fn` to `creator` as `grad_fn` throws error out.

Paste my error info below.

```
Traceback (most recent call last):
  File "./network.py", line 56, in <module>
    print(loss.creator.next_functions[0][0])
AttributeError: 'MSELoss' object has no attribute 'next_functions'
```

**Comment by chsasank** [comment]
This is related to PR #79. See my comment [here](https://github.com/pytorch/tutorials/pull/93#issuecomment-304780837).

tldr: changes in master pytorch.
creator -> grad_fn
previous_functions -> next_functions

**Comment by apaszke** [comment]
We shouldn't change tutorials (especially beginner ones) to work with master. They should always be prepared for the last release

**Comment by chsasank** [comment]
I agree. Previously pytorch docs used to be built with master rather than release. Following the same, PR #79 was merged. (When the PR was created, docs already had `grad_fn` and `next_functions`.)

In the future, we'll keep a branch for all compatibility breaking changes and merge it after the release.
